ruby_on_rails/configure_cd.md

```diff
@@ -4,26 +4,36 @@ We use Semaphore to deploy most of our apps. Some projects use CircleCI and
 mobile apps use Fastlane. This article is about how to deploy to Heroku with
 Semaphore.
 
-## Prerequisites
+## Setup `master`, `develop` and `testing`
 
-* Heroku API key of `admin@renuo.ch` (*wg-operations* can help with that)
+* Add the following to your `semaphore.yml`:
```

Comment:
Where is this file located?

Reply:
It is located under the `.semaphore` directory in the project root, as [described here](https://github.com/renuo/applications-setup-guide/blob/semaphore-2.0/ruby_on_rails/configure_ci.md#L16)